pkgs/applications/graphics/litemdview/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, fetchpatch
+, gtkmm3
+, autoconf
+, automake
+, libtool
+, cairo
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "litemdview";
+  version = "eacbbc837e";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "g0tsu";
+    repo = "litemdview";
+    rev = "eacbbc837e";
+    hash = "sha256-sD0y+/7TogywrvftxEXT1EyBf3BXH9QWbawFVOF8xRc=";
+  };
+
+  buildInputs = [
```

Comment:
Some of your current `buildInputs` should be `nativeBuildInputs`, on a first glance at least `autoconf`, `automake`, `pkg-config`.